governance-aepp/src/views/Create.vue

```diff
@@ -79,13 +90,13 @@
       </div>
       <div class="text-gray-500 text-sm p-2">
         <span v-if="closeHeight && closeHeight > height">
-          To create a never closing poll, set close height to 0.
+          To create a poll that runs infinitely, set "Close at height" to 0.
```

Comment:
why text change?

Reply:
Text changes are requested by the communications team.